src/library/DIPS.Mobile.UI/Resources/Styles/Label/LabelStyle.cs

```diff
@@ -17,5 +17,9 @@ public enum LabelStyle
     Header700=12,
     Header600=13,
     Header500=14,
-    SectionHeader=15
+    SectionHeader=15,
+    KeyOverValue=16,
```

Comment:
Ser ser ikke sammenheng med disse stilene og en label. En label har en tekst, hva betyr da i så fall `KeyOverValue` , `ValueBelowKey`, `KeyInLineWithValue`og `ValineInLineWithKey` ? Dette virker som stiler som er spesifikt for ListItem.